public/index.html

```diff
@@ -5,7 +5,11 @@
     <title>Google Shopping</title>
   </head>
   <body>
+    <div id="productImagesService"></div>
     <div id="reviews-service"></div>
+    <div id="app"></div>
```

Comment:
I changed my div id to product-information I would update this

Reply:
This div was for a different service (Seller). Also, I did not see in your branches that your component was was being appending to product-information but I have made the change.